python/ray/serve/tests/test_deploy_app.py

```diff
@@ -685,21 +685,30 @@ def test_update_config_graceful_shutdown_timeout(client: ServeControllerClient):
     )
 
 
-def test_update_config_max_concurrent_queries(client: ServeControllerClient):
-    """Check that replicas stay alive when max_concurrent_queries is updated."""
+@pytest.mark.parametrize("use_max_concurrent_queries", [True, False])
```

Comment:
Same for this would be nice to test when both are set and when neither are set

Reply:
Also, would be nice to set them to different values and checking the override behavior 🙃

Reply:
Also, differ from default to check the default behavior